schemas/v1.0/azure.yaml.json

```diff
@@ -148,6 +148,22 @@
                 },
                 "required": ["project"]
             }
+        },
+        "pipeline": {
+            "type": "object",
+            "title": "Definition of continuous integration pipeline",
+            "additionalProperties": true,
```

Comment:
Any specific reason why we are supporting additional properties here? Do custom properties get passed down into the providers today?

Reply:
copied from bicep.. I will remove it.  We don't have custom properties